dev-requirements.txt

```diff
@@ -1,6 +1,5 @@
 percy
 selenium
 pandas
-dash_table_experiments
 xlrd
```

Comment:
Changed table experiment for the table while at it -- the experiment was causing problems in any case

Reply:
Just for the tests, or is the transition to 16 the end of the line for anyone still using experiments? Which would be fine, but it would mean we need to be very clear about this with users.

Reply:
I changed this because I was having issues with the experiment. Will check if this is end-of-line or just usage details.

Reply:
As discussed, will not looking further into this.